gossip/privdata/reconcile.go

```diff
@@ -316,3 +316,29 @@ func (r *Reconciler) groupRwsetByBlock(elements []*protosgossip.PvtDataElement)
 	}
 	return rwSetByBlockByKeys
 }
+
+func constructUnreconciledMissingData(requestedMissingData privdatacommon.Dig2CollectionConfig, fetchedData []*protosgossip.PvtDataElement) ledger.MissingPvtDataInfo {
+	remainingRequestedData := requestedMissingData
```

Comment:
The name of the function does not indicate that the original input will be modified. If that's the intention, I would suggest to call it this function as `removeFetchedData` and do not return anything - instead let the caller prepared the `MissingPvtDataInfo` object.

Reply:
Yes. It does not sound good to manipulate the requested data. I have avoided removing entries from the requested data while constructing the unreconciled missing data.